src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -62,7 +62,7 @@ static object OnIsRefreshingPropertyCoerced(BindableObject bindable, object valu
 		public bool IsRefreshing
 		{
 			get { return (bool)GetValue(IsRefreshingProperty); }
-			set { SetValue(IsRefreshingProperty, value); }
+			set { SetValue(IsRefreshingProperty, value, SetterSpecificity.FromHandler); }
```

Comment:
There are a few scenarios of this that we are currently fixing. 

Use an explicit interface to use a different `SetterSpecificity` for values coming from the handler. 

https://github.com/dotnet/maui/pull/17036/files#diff-a636f91eba00da8fba0f0c86b5ec4274498294234e5597180eddf1fae6f5116bR269-R273

Reply:
this is wrong. the Handler should set the BP directly, not the c# property, with the right specificity